x/oracle/types/events.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+const (
+	EventTypeOracleRegistration = "oracle_registration"
```

Comment:
For the name of this event type, how about `EventTypeRegistrationVote`?
When we see other event types, it seems to refer the detailed action of each message.

Reply:
Right. It seems good.
I fixed it. (https://github.com/medibloc/panacea-core/pull/391/commits/324e15294580b56398d52874e55c3ca2aa0f84cc)